### ISSUE TITLE
Change structure to resemble actual wiki's

### ISSUE THREAD

**Issue by democat3457** [DeltaDizzy/Kop-Wiki-Backup#1]
I feel like it might be more intuitive to structure the backup repo's files into the same way they are organized in [the wiki's "All Pages" section.](https://kopernicus.tmsp.io/all) Plus, it makes the .md links work in the repo.

**Comment by DeltaDizzy** [comment]
The bigger issue is how to get the home page to display. GitHub can do it but we would have to name the file "readme.md" and it would be underneath the file list. Personally, I think we should wait until the wiki is fully migrated before deciding on what format we want to go with, whether its Pages, Wiki (both on GitHub), or DokuWiki (as Thomas and Jade are speculating about).

**Comment by democat3457** [comment]
You could move the info in README.md to the description of the repo and remove the README entirely (as it is not needed now). But yes, I agree with the waiting on the wiki format.

**Comment by DeltaDizzy** [comment]
Closing this and opening a new one for the wiki format.
